src/components/PostList/PostList.jsx

```diff
@@ -1,3 +1,10 @@
-export const PostList = () => (
-  <>Put the list here</>
+import React from 'react';
+import { PostInfo } from '../PostInfo';
+
+export const PostList = ({ posts }) => (
+  <>
```

Comment:
Use semantic tags for list `ul` `li`. Don't forget to remove default `li` dots and `ul` paddings
```suggestion
  <ul>
```